### PR TITLE
Allow deploying of all types for performance beta

### PR DESCRIPTION
**Pull request by brpowell**
### What does this PR do?

Enables deploying of all types for deploy performance beta.

### What issues does this PR fix or reference?
@W-8090826@

### CODE REVIEW

File: docs/_articles/en/user-guide/perf-enhancements.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ To enable this beta feature:
 1. Select **File** > **Preferences** > **Settings** (Windows or Linux) or **Code** > **Preferences** > **Settings** (macOS).
 1. Under Salesforce Feature Previews, select Experimental: Deploy Retrieve.
 
-In this beta release, performance enhancements are effective when you run the **SFDX: Deploy This Source to Org** and **SFDX: Retrieve This Source from Org** command with these metadata types:
+In this beta release, performance enhancements are effective when you run the **SFDX: Deploy This Source to Org** and **SFDX: Retrieve This Source from Org** commands. At this time, the beta enhancements for **SFDX: Retrieve This Source from Org** only works with the following metadata types:
```
Comment (brpowell) [inline docs/_articles/en/user-guide/perf-enhancements.md:17]: @ekapner doc update with the change

File: packages/salesforcedx-vscode-core/package.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
     "@salesforce/apex-node": "0.0.9",
     "@salesforce/core": "2.9.0",
     "@salesforce/salesforcedx-utils-vscode": "49.10.0",
-    "@salesforce/source-deploy-retrieve": "1.0.20",
+    "@salesforce/source-deploy-retrieve": "1.1.2",
```
Comment (lcampos) [inline packages/salesforcedx-vscode-core/package.json:31]: Shouldn't we use version `1.1.3` now ?

**Review by lcampos — APPROVED** [review]

File: docs/_articles/en/user-guide/perf-enhancements.md (hunk 1 above)
Comment (ekapner) [reply docs/_articles/en/user-guide/perf-enhancements.md:17]: @brpowell - small tweaks (works to work, move the "only", remove "the following" to something more conversational):

At this time, the beta enhancements for **SFDX: Retrieve This Source from Org** work with only these metadata types:

**Review by ekapner — COMMENTED** [review]
Minor tweaks to wording of performance enhancements message

**Review by AnanyaJha — APPROVED** [review]

**Pull request merged**